### PR TITLE
Feature ID visibility added

### PR DESCRIPTION
**Pull request by shweta2101**
1. Feature Id is made visible for stations and services.
2. Added clipboard functionality to copy Id.

### CODE REVIEW

**Comment by shweta2101** [comment]
Screenshot 2023-09-11 at 3 33 36 PM

File: tdei-ui/src/routes/Services/ClipBoardCopy.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import copyIcon from "../../assets/img/copy-icon.svg";
+import copiedIcon from "../../assets/img/clipboard-copied.svg"
+///
+/// Copy to ClipBoard Component
+///
+function ClipboardCopy({ copyText }) {
```
Comment (MashB) [inline tdei-ui/src/routes/Services/ClipBoardCopy.js:7]: I see this is new component, but please check we have already added plugin for copyclipboard and using in below file tdei-ui/src/components/UserHeader/UserHeader.js .

**Review by MashB — CHANGES_REQUESTED** [review]

File: tdei-ui/src/routes/Services/ClipBoardCopy.js (hunk 1 above)
Comment (shweta2101) [reply tdei-ui/src/routes/Services/ClipBoardCopy.js:7]: Component is present but it is behaving differently. Tried integrating the same here also but facing css issues. Attaching screenshot
Screenshot 2023-09-11 at 4 18 21 PM

Comment (shweta2101) [reply tdei-ui/src/routes/Services/ClipBoardCopy.js:8]: Updated code with 'copyclipboard'. Please check the changes.

**Review by MashB — APPROVED** [review]

**Pull request merged**